src/plugins/intel_cpu/src/emitters/snippets/aarch64/kernel_executors/gemm_copy_b.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "common/utils.hpp"
+#include "cpu_memory.h"
+#include "emitters/snippets/brgemm_generic.hpp"
+#include "emitters/utils.hpp"
+#include "kai/ukernels/matmul/matmul_clamp_f32_f32_f32p/kai_matmul_clamp_f32_f32_f32p8x1biasf32_6x8x4_neon_mla.h"
+#include "kai/ukernels/matmul/matmul_clamp_f32_f32_f32p/kai_matmul_clamp_f32_f32_f32p_interface.h"
+#include "kai/ukernels/matmul/pack/kai_rhs_pack_kxn_f32p8x1biasf32_f32_f32_neon.h"
+#include "transformations/snippets/aarch64/pass/lowered/gemm_cpu_blocking.hpp"
+
+namespace ov::intel_cpu::aarch64 {
+
+struct GemmCopyBKernelKaiConfig : public snippets::KernelExecutorBase::GenericConfig {
+public:
+    GemmCopyBKernelKaiConfig() = default;
+    GemmCopyBKernelKaiConfig(const size_t n_blk_size);
+
+    GemmCopyBKernelKaiConfig& operator=(GemmCopyBKernelKaiConfig other);
+
+    bool operator==(const GemmCopyBKernelKaiConfig& rhs) const;
+    bool operator!=(const GemmCopyBKernelKaiConfig& rhs) const {
+        return !(*this == rhs);
+    }
+
+    [[nodiscard]] std::unique_ptr<GenericConfig> get_clone_ptr() const override {
+        return std::make_unique<GemmCopyBKernelKaiConfig>(*this);
+    }
+
+    [[nodiscard]] bool is_completed() const override;
+
+#ifdef SNIPPETS_DEBUG_CAPS
+    virtual std::string to_string() const override;
+#endif
+
+    void update(size_t N, size_t K);
+
+    size_t hash() const override {
+        return m_hash;
+    }
+
+    [[nodiscard]] size_t get_N() const {
+        return m_N;
+    }
+    [[nodiscard]] size_t get_K() const {
+        return m_K;
+    }
+    [[nodiscard]] size_t get_n_blk_size() const {
+        return m_n_blk_size;
+    }
+
+private:
+    [[nodiscard]] size_t compute_hash() const;
+
+    size_t m_N = 0;
+    size_t m_K = 0;
+    const size_t m_n_blk_size = 0;
+    size_t m_hash{SIZE_MAX};
+};
+
+struct GemmCopyBCompiledKernel {
+    std::shared_ptr<kai_matmul_clamp_f32_f32_f32p_ukernel> copy_b_ukernel =
+        std::make_shared<kai_matmul_clamp_f32_f32_f32p_ukernel>(ukernel);
+    std::shared_ptr<std::vector<uint8_t>> bias_buffer = std::make_shared<std::vector<uint8_t>>();
```

Comment:
Minor:
```suggestion
    std::shared_ptr<std::vector<uint8_t>> bias_buffer = {};
```

Reply:
1. u8 is used just for other precision support in future.
2. I would like just include two shared_ptr inside a small structure. the the object ukernel and memory managed separated. seem like the two are more aligned and more understandable for me:-)